docs/integrations/parameter-types.md

```diff
@@ -5,13 +5,15 @@ sidebar_label: Parameter Types
 ---
 
 Integration Parameter Types are used to configure instances of the integration. When adding a parameter to an integration in Cortex XSOAR, there are numerous types to choose from.
-Each type will affect the parameter behavior and interaction with the user. 
+Each type will affect the parameter behavior and interaction with the user.
+Please see [here](https://xsoar.pan.dev/docs/integrations/yaml-file#configuration) for more information how to set the parameter type.
 
 ## Boolean
 
 This type of parameter creates a check box in the integration configuration. When the check box is ticked, the value in
 the integration code would be `True`, and `False` otherwise.
-Example:
+The type number is: 0.
```

Comment:
I removed the example since it's not looks good.
https://xsoar.pan.dev/docs/integrations/parameter-types#boolean